pkgs/development/python-modules/strpdatetime/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  textx,
+  pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "strpdatetime";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "RhetTbull";
+    repo = "strpdatetime";
+    rev = "121d8ef1b3d7c4dcfe012da94810fe83d869d748";
```

Comment:
Please ask upstream to add a tag.

Reply:
There are tag, but not for the 0.3.0 version, will ask to add one

Reply:
Tag exists now